.github/workflows/cron_nightly_build.yml

```diff
@@ -70,6 +70,11 @@ jobs:
         run: today=`date +%Y-%m-%d-`; for i in *; do mv $i $today$GH_BRANCH-$i; done
         working-directory: ${{ env.RELEASE_DIR }}
 
+      - name: Create nightly.zip for develop
+        if: matrix.BRANCH == 'develop'
+        run: for i in *; do cp $i nightly.zip; done
```

Comment:
Why do we needs a loop over files here? At the end, only the last item will be copied to nightly.zip right?

Reply:
I thought it will be easier if we have one file, but I'm wrong, their is an xml file too, So I will update that.

Reply:
done